vmdb/lib/miq_automation_engine/service_models/miq_ae_service_vm_or_template.rb

```diff
@@ -1,5 +1,8 @@
 module MiqAeMethodService
   class MiqAeServiceVmOrTemplate < MiqAeServiceModelBase
+    require_relative "mixins/miq_ae_service_ems_operations_mixin"
```

Comment:
@kbrock 
The reason I added it under the class was because the mixin has an instance method. If the include is outside the class scope would the method be still an instance method